components/homme/src/share/cxx/scream_kokkos_utils.hpp

```diff
@@ -0,0 +1,189 @@
+#ifndef SCREAM_KOKKOS_UTILS_HPP
+#define SCREAM_KOKKOS_UTILS_HPP
+
+#include "Types.hpp"
+#include "ExecSpaceDefs.hpp"
+
+#include "../../algorithms/src/Kokkos_Random.hpp"
+
+#include <chrono>
+#include <cassert>
+#include <type_traits>
+
+// This file should not be merged with share/scream_kokkos_meta.hpp.
```

Comment:
This comment should probably be deleted, cause it refers to a possible misunderstanding of files in the scream repo. Here, we don't have those files, so this is confusing at best.

Reply:
Done.